protocol/flatbuffers/rpc.fbs

```diff
@@ -27,7 +27,10 @@ union RpcMessage {
 	OpenSerialRequest,
 	CloseSerialRequest,
 	SetWifiRequest,
-	SerialUpdateResponse
+	SerialUpdateResponse,
+	AutoBoneProcessRequest,
+	AutoBoneProcessStatus,
```

Comment:
This should have `Response` suffix. I'm assuming here that this is a response to `AutoBoneProcessRequest`. Are there multiple Responses for each request? If so, this is not an appropriate use of the RPC system.

Reply:
> Are there multiple Responses for each request? If so, this is not an appropriate use of the RPC system.
It is a response, so there are multiple responses, much the same as `SerialUpdateResponse` which I had based it off.

Reply:
OK, after discord conversation, just update to have `Response` in the name